mnefun/_scoring.py

```diff
@@ -117,12 +118,18 @@ def _read_events(p, subj, ridx, raw, picker=None):
     if picker == 'restrict':  # limit to events that will be processed
         ids = p.in_numbers
         picker = None
-        print('Events restricted to those in params.in_numbers')
+        print('    Events restricted to those in params.in_numbers')
     else:
         ids = None
     events = list()
     for fname in get_event_fnames(p, subj, ridx):
-        these_events = read_events(fname, include=ids)
+        # gracefully handle empty events (e.g., resting state)
+        with open(fname, 'r') as fid:
+            content = fid.read().strip()
+        if not content:
+            these_events = np.empty((0, 3), int)
+        else:
+            these_events = read_events(fname, include=ids)
```

Comment:
Added this to handle writing an empty events list (which should be supported, for example when doing a resting-state run that will be analyzed separately)

Reply:
I double checked that an empty events handed to `concatenate_events` is acceptable, so I think we're good to go.